### PR TITLE
Fix external Outbound Transport loading code

### PR DESCRIPTION
**Pull request by frostyfrog**
As the code currently stands, only internal outbound transports are able to be loaded. With these changes, it should be possible to load a new outbound transport via a plugin.

### CODE REVIEW

File: aries_cloudagent/transport/outbound/manager.py [hunk 1]
```diff
@@ -117,13 +117,19 @@ def register(self, module: str) -> str:
...
+                module = module_name
+
             imported_class = ClassLoader.load_subclass_of(
-                BaseOutboundTransport, module, MODULE_BASE_PATH
+                BaseOutboundTransport, module, package
             )
-        except (ModuleLoadError, ClassNotFoundError):
-            raise OutboundTransportRegistrationError(
+        except (ModuleLoadError, ClassNotFoundError) as e:
+            raise InboundTransportRegistrationError(
```
Comment (shaangill025) [inline aries_cloudagent/transport/outbound/manager.py:130]: Shouldn't this be `Outbound`?

Comment (frostyfrog) [reply aries_cloudagent/transport/outbound/manager.py:130]: It should indeed. Great catch! 😄

**Comment by shaangill025** [comment]
@frostyfrog Is this PR ready for review? I think this PR should be part of `0.7.4` release. Let me know if I can help with anything.

**Review by andrewwhitehead — APPROVED** [review]

**Comment by swcurran** [comment]
@frostyfrog -- can you please update the PR to include the base branch.  Please watch for other merges in the next hour or so, and let me know when you provide the update, so I can merge this before I merge others where I can update the base myself.

**Comment by frostyfrog** [comment]
@swcurran Feel free to give me another ping if needed. My apologies that this PR didn't grant permissions. My mistake

**Comment by swcurran** [comment]
No prob...I'm watching it.... :-)

**Pull request merged**